src/ResizableBox.tsx

```diff
@@ -100,6 +101,9 @@ const ResizeHandle: React.FC<{
       onPointerUp={() => {
         send(dragDropModel.events['DRAG.END']());
       }}
+      onPointerCancel={() => {
+        send(dragDropModel.events['DRAG.END']());
+      }}
```

Comment:
for some reason `cy.realSwipe` was finishing with a pointercancel event and not pointerup, unsure why that is but handling pointercancel here is a good idea either way

Reply:
That's weird. In real life, how would a pointercancel event occur?

Reply:
From the https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/pointercancel_event the most realistic ones for us are:

> A hardware event occurs that cancels the pointer activities. This may include, for example, the user switching applications using an application switcher interface or the "home" button on a mobile device.

and:

> The device's screen orientation is changed while the pointer is active.

I was able to trigger it only by selecting a text and dragging it outside of the current window. Funny enough - it didn't actually cancel the text dragging session, I was still able to drop it successfully onto the URL bar. 🤷‍♂️ 

No harm in handling this since it exists though - I agree though that I would prefer to know when this can happen in real life 😅